docs/_build/html/_sources/docker_compose.md.txt

```diff
@@ -0,0 +1,25 @@
+# Docker Compose
+Tilt can use `docker-compose` to orchestrate your services (instead of Kubernetes). This doc describes how you can get Tilt's UX for your Docker Compose project with a one-line Tiltfile. This is simpler than the  config for Kubernetes projects described in the [Tutorial](tutorial.html).
+
+
+## Comparison
+Tilt provides a better User Experience in two ways:
+* Tilt's UI shows you status at a glance, so errors can't scroll off-screen. You can navigate the UI in your terminal and dig into the logs for just one service. (Tilt also has a global log if you do want the full firehose).
+* Tilt handles filesystem watching and updating without requiring manual actions or hand-rolled scripting.
+
+Most of our documentation describes using Tilt to deploy to Kubernetes, but that would be a large change. For Docker Compose projects, Tilt uses Docker Compose as a backend. This allows you to use your existing configuration, debugging tricks, and muscle memory while getting a better UX.
```

Comment:
the change that I was getting at here is that "but that would be a large change" doesn't read -- i.e. you're talking about changing your project to k8s just to use tilt but the "that" doesn't clearly reference that.